include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -523,6 +524,12 @@ class NBL_API2 IGPUCommandBuffer : public IBackendObject
         };
         bool resolveImage(const IGPUImage* const srcImage, const IGPUImage::LAYOUT srcImageLayout, IGPUImage* const dstImage, const IGPUImage::LAYOUT dstImageLayout, const uint32_t regionCount, const SImageResolve* const pRegions);
 
+        bool IGPUCommandBuffer::traceRays(const asset::SStridedBufferRegion<IGPUBuffer>& raygenGroupRegion,
+          const asset::SStridedBufferRegion<IGPUBuffer>& missGroupsRegion,
+          const asset::SStridedBufferRegion<IGPUBuffer>& hitGroupsRegion,
+          const asset::SStridedBufferRegion<IGPUBuffer>& callableGroupsRegion,
+          uint32_t width, uint32_t height, uint32_t depth);
```

Comment:
The strided buffer it better expressed as bufferbinding (buffeR+offset struct) and a stride as a separate thing

P.S. what about `traceRaysIndirect` and the command that sets the stack size?